website1/src/scss/_base.scss

```diff
@@ -0,0 +1,32 @@
+html,
+body {
+  height: 100%;
+  font-family: "Source Sans Pro", sans-serif;
+  font-weight: 400;
+  font-size: 18px;
+  color: $font-color-grey;
+  .wrapper {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    .container {
+      width: $w-100;
+      max-width: 1290px;
+    }
+    .mx-auto {
+      margin-left: auto;
+      margin-right: auto;
+    }
+    .section {
+      width: $w-100;
+      display: flex;
+    }
+    a {
+      text-decoration: none;
+      color: $font-color-grey;
+    }
+    .text-bold {
+      font-weight: 700;
+    }
+  }
```

Comment:
Here need turn back, because it's universal classes, they should not be prioritized.